src/explore-education-statistics-admin/src/pages/publication/__tests__/PublicationDetailsPage.test.tsx

```diff
@@ -70,25 +70,54 @@ describe('PublicationDetailsPage', () => {
       id: 'publication-1',
       slug: 'publication-1-slug',
       title: 'Publication 1',
+      owner: false,
+      contact: {
+        teamName: 'Mock Contact Team Name',
+        teamEmail: 'Mock Contact Team Email',
+        contactName: 'Mock Contact Name',
+      },
     },
     {
       id: 'publication-2',
       slug: 'publication-2-slug',
       title: 'Publication 2',
+      owner: false,
+      contact: {
+        teamName: 'Mock Contact Team Name',
+        teamEmail: 'Mock Contact Team Email',
+        contactName: 'Mock Contact Name',
+      },
     },
     {
       id: 'publication-3',
       slug: 'publication-3-slug',
       title: 'Publication 3',
+      owner: false,
+      contact: {
+        teamName: 'Mock Contact Team Name',
+        teamEmail: 'Mock Contact Team Email',
+        contactName: 'Mock Contact Name',
+      },
     },
   ];
 
+  const mockPublication: Publication = {
```

Comment:
Would name this `testPublication` for consistency

Reply:
Have done.